Xpense/urls.py

```diff
@@ -14,10 +14,10 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 from main import views
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', views.welcome_page, name='welcome_page'),
+    path('test/', include('house.urls'))
```

Comment:
```
from house.views import dashboard  
path('house/', dashboard, name="dashboard")
```

I would even do that in 
```
path('', dashboard, name="dashboard")
```
because now it looks like that:
![image](https://user-images.githubusercontent.com/57755873/162607032-4522ce89-0d8b-4bd6-9219-a84300a474cc.png)